benchmarks/agieval/agieval_cloze.py

```diff
@@ -0,0 +1,11 @@
+from agieval import task_template
+
+from inspect_ai import task
+
+
+## Cloze Tasks
```

Comment:
It might make sense to move this task to `agieval_en.py` since it uses the EN dataset.

Reply:
I thought I would be easier to keep it in a separate file as the AGIEval_en in the paper consist in all MCQ tasks in english. It allows to run them more easily as they are in the same file. 

I am inclined to think it is more easy to keep it this way but happy to move it if needed.